recipes/pyscrap/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: pyscrap
+  version: '0.0.9'
+
+source:
+  fn: pyscrap-0.0.9.tar.gz
+  url: https://pypi.python.org/packages/aa/cb/07ac450abb5c2eb7cdf52ceb394c8a35e4cca440e59e964a029d0696f181/pyscrap-0.0.9.tar.gz
+  md5: 9b4ec62d2383e199724903b580b6ea4b
+  patches:
+    - import.patch  # [py3k]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - simplejson
+    - lxml
+    - libxml2
+    - libxslt
+
+  run:
+    - python
+    - simplejson
+    - lxml
+    - libxml2
+    - libxslt
+
+test:
+  imports:
+      - pyscrap
+
+about:
+  home: https://github.com/Zincr0/pyscrap
+  license: Apache-2.0
```

Comment:
This needs a license file and a license family

Reply:
The license file is only in the repo, and not part of tar.gz.

Reply:
https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
Ah, thank you @CJ-Wright !